tests/integration/workspace_access/test_groups.py

```diff
@@ -60,17 +60,49 @@ def test_rename_groups(ws, make_ucx_group, sql_backend, inventory_schema):
 
 
 @retried(on=[NotFound], timeout=timedelta(minutes=2))
-def test_reflect_account_groups_on_workspace_skips_groups_that_already_exists_in_the_workspace(
+def test_reflect_account_groups_on_workspace_warns_skipping_when_a_workspace_group_has_same_name(
     caplog,
     ws,
     make_ucx_group,
     sql_backend,
     inventory_schema,
 ):
-    """The groups that already are reflected in the workspace should be skipped."""
+    """Warn about groups for which a workspace group with the same name exists."""
     ws_group, acc_group = make_ucx_group(wait_for_provisioning=True)
 
     group_manager = GroupManager(sql_backend, ws, inventory_schema, [ws_group.display_name], "ucx-temp-")
+    with caplog.at_level(logging.WARN, logger="databricks.labs.ucx.workspace_access.groups"):
+        group_manager.reflect_account_groups_on_workspace()
+    assert f"Skipping {acc_group.display_name}: group already exists in workspace" in caplog.text
```

Comment:
As mentioned elsewhere, I think this should be a hard error.

Even if we decided against raising an error, the terminology here is wrong: we're more than warning about the problem, we're treating it as an error situation.